src/pages/demo/storyListing/index.tsx

```diff
@@ -0,0 +1,88 @@
+import { drupalClient } from '@/utils/drupalClient'
+import { GetStaticPropsContext, GetStaticPropsResult } from 'next'
+import StoryListing from '@/components/node/story_listing'
+import { NodeNewsStory, NodeStoryListing } from '@/types/node'
+import { filter, forEach } from 'lodash'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+
+interface StoryListingPageProps {
+  nodeStoryListings: NodeStoryListing
+  nodeNewsStoryTeasers: NodeNewsStory
+}
+
+const StoryListingPage = ({
+  nodeStoryListings,
+  nodeNewsStoryTeasers,
+}: StoryListingPageProps) => {
+  return (
+    <>
+      <StoryListing
+        nodeStoryListings={nodeStoryListings}
+        nodeNewsStoryTeasers={nodeNewsStoryTeasers}
+      />
+    </>
+  )
+}
+
+export default StoryListingPage
+
+function filterNewsStoryTeasersById(nodeStoryListings, nodeNewsStoryTeasers) {
+  let matchingNewsStories: NodeNewsStory = null
+  const storyListing = nodeStoryListings[0]
+
+  const storyListingId = storyListing.drupal_internal__nid
+
+  matchingNewsStories = filter(nodeNewsStoryTeasers, function (newsStory) {
+    const newsStoryTeaserId =
+      newsStory.field_listing.resourceIdObjMeta.drupal_internal__target_id
+
+    return newsStoryTeaserId === storyListingId
+  })
+
+  return matchingNewsStories
+}
```

Comment:
The way the data is currently being fetched in the new story listing component we won't need filtering at this time.